webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/connectome/Hdf5ConnectomeFileService.scala

```diff
@@ -43,13 +43,16 @@ class Hdf5ConnectomeFileService @Inject()(config: DataStoreConfig) extends FoxIm
       fromAndToPtr: Array[Long] <- finishAccessOnFailure(cachedConnectomeFile) {
         cachedConnectomeFile.uint64Reader.readArrayBlockWithOffset(keyCscIndptr, 2, agglomerateId)
       } ?~> "Could not read offsets from connectome file"
-      from <- fromAndToPtr.lift(0).toFox ?~> "Could not read start offset from connectome file"
-      to <- fromAndToPtr.lift(1).toFox ?~> "Could not read end offset from connectome file"
+      fromPtr <- fromAndToPtr.lift(0).toFox ?~> "Could not read start offset from connectome file"
+      toPtr <- fromAndToPtr.lift(1).toFox ?~> "Could not read end offset from connectome file"
+      _ <- Fox.fromBool(toPtr >= fromPtr) ?~> s"Agglomerate $agglomerateId not present in agglomerate file"
```

Comment:
Very nitpicky, but maybe rewrite the assertion to read more natrually IMO? Your commit message has the same way of expressing the assertion showing that this might be the more intuitive version:
```suggestion
      _ <- Fox.fromBool(fromPtr <= toPtr) ?~> s"Agglomerate $agglomerateId not present in agglomerate file"
```